lib/js/jsconsole.nim

```diff
@@ -42,28 +43,34 @@ macro info*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]):
 macro error*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
   makeConsoleCall(console, bindSym "errorImpl", args)
 
+macro exception*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  makeConsoleCall(console, bindSym "errorImpl", args)
+
+macro trace*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  makeConsoleCall(console, bindSym "traceImpl", args)
 
 macro warn*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
-  ## https://developer.mozilla.org/en-US/docs/Web/API/Console/warn
```

Comment:
same comment as in earlier PR: why delete those useful links? having clickable links in docs helps

Reply:
https://github.com/nim-lang/Nim/pull/12455#discussion_r336640434

Reply:
consensus for this PR https://github.com/nim-lang/Nim/pull/13483 suggests that we should keep them, eg see https://github.com/nim-lang/Nim/pull/13483#discussion_r389200052
That PR ended up keeping all doc comments. All you need is to make the line not too long, eg:
```nim
proc clear*(console: Console) {.importcpp: "clear".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/clear
proc clear*(console: Console) {.importcpp: "clear".}
  ## https://developer.mozilla.org/en-US/docs/Web/API/Console/clear
```
These doc comments are valuable, and makes the upstream doc 1 click away.

Reply:
Done.